### DISCUSSION TITLE
Google fonts import not working in v.3.0.1 on Vercel

### DISCUSSION THREAD

**Discussion by Existenziell** [tailwindlabs/tailwindcss#6467] [Help] [answered]
I used to be able to simply import a Google font in my globals.css and set it in tailwind.config.js. 
Since the update to tailwind 3.0.1 this doesn't work any longer, at least not after deploying to Vercel. There is not even a network call towards static.google.fonts happening. Anyone an idea what could be the issue here?

globals.css:

@import url("https://fonts.googleapis.com/css?family=Gotu&display=swap");

@tailwind base;
@tailwind components;
@tailwind utilities;

tailwind.config.js:
theme: {
    extend: {
      fontFamily: {
        sans: [
          'Gotu',
        ],
      },
   },
}

**Comment by Existenziell** [comment, marked as answer]
Nevermind, this was fixed in 3.0.2.
I would like to follow-up with the general question: 
What actually is the best practice for importing Google fonts into tailwind?

**Reply by Existenziell** [reply]
Ok, sorry for that... 
This answers everything: https://web.dev/font-best-practices/
TL;DR: link in Head is preferred combined with preconnect

**Reply by raihan-suryanom** [reply]
This issue occurs again in version 3.0.18 (latest)
